Packages/Tracking/OpenXR/Runtime/Scripts/OpenXRLeapProvider.cs

```diff
@@ -220,14 +220,23 @@ private bool PopulateLeapHandFromOpenXRJoints(HandTracker handTracker, ref Hand
                         continue;
                     }
 
+                    var prevJointPosition = prevJoint.Pose.position;
+                    var nextJointPosition = nextJoint.Pose.position;
+                    
+                    // Adjust the fingertip positions forward by a radius to match LeapC data.
+                    if (boneIndex == 3)
+                    {
+                        nextJointPosition += prevJoint.Pose.forward * nextJoint.Radius;
```

Comment:
Is the capsule size equal to the radius of the joint? I thought the radius is configurable in the editor. Should it not be used here?